packages/synapse-react-client/src/utils/hooks/useConfirmItems.test.ts

```diff
@@ -0,0 +1,78 @@
+import { renderHook, act } from '@testing-library/react'
+import useConfirmItems from './useConfirmItems'
+
+describe('useConfirmItems', () => {
```

Comment:
fyi, the tests in this suite were ~90% written by GitHub Copilot, with minor fixes

Reply:
I asked ChatGPT review, and it said it looks ok ;)